charts/kong/templates/_helpers.tpl

```diff
@@ -95,33 +95,75 @@ metadata:
     {{- end }}
   {{- end }}
 spec:
-{{- if (and (not (eq .ingressVersion "extensions/v1beta1")) .ingress.ingressClassName) }}
+{{- if .ingress.ingressClassName }}
   ingressClassName: {{ .ingress.ingressClassName }}
 {{- end }}
   rules:
+  {{- if ( not (or $hostname .ingress.hosts)) }}
+  - host: {{ $hostname | quote }}
```

Comment:
Nit: as here we already know that `$hostname` is empty, we might as well omit defining `host` field.

Reply:
Ofc! Will fix...